packages/eden/src/client.ts

```diff
@@ -12,12 +12,12 @@ import type {
 } from './links/internal/operation'
 import { share } from './links/internal/operators'
 import type { EdenResponse } from './request'
-import type { EdenRequestParams } from './resolve'
+import type { EdenRequestParams as EdenRequestParameters } from './resolve'
```

Comment:
I don't see the benefit of locally renaming this type. There are no name conflicts, and if the preference is to use fully verbalized names, then the original type should be adjusted.

This applies to all future instances of renaming this type.